editor/editor_audio_buses.cpp

```diff
@@ -1337,7 +1337,8 @@ EditorAudioBuses::EditorAudioBuses() {
 
 	bus_scroll = memnew(ScrollContainer);
 	bus_scroll->set_v_size_flags(SIZE_EXPAND_FILL);
-	bus_scroll->set_vertical_scroll_mode(ScrollContainer::SCROLL_MODE_DISABLED);
+	bus_scroll->set_custom_minimum_size(Size2(0, 200) * EDSCALE);
+	bus_scroll->set_size(Size2(0, 350) * EDSCALE);
```

Comment:
I don't think `set_size()` has any effect here.
![image](https://github.com/user-attachments/assets/873b3947-c783-4678-bdfe-0c9d2c4772b5)
Minimizing the editor panels should not affect editor at full size. We need a better sizing mechanism before doing such changes.

Reply:
I think I have an idea about it. I will try it and then write the results here.

Reply:
@KoBeWi, I made this https://github.com/godotengine/godot/pull/109915. I think in theory it can be used in all other editors too.